src/Illuminate/Container/Container.php

```diff
@@ -1058,8 +1059,16 @@ public function build($concrete)
 
         array_pop($this->buildStack);
 
+        if (!empty($reflector->getAttributes(Lazy::class))) {
```

Comment:
You could just directly check this
```suggestion
        if ($reflector->getAttributes(Lazy::class) !== []) {
```

Reply:
Not much. Technically, a function call is more expensive than a comparison, but this should mostly be optimized away in practice. One _slight_ advantage would be a clear communication that we are working with an array here and don't have to check _any_ structure's emptiness.